new-lamassu-admin/src/pages/Analytics/graphs/TopMachinesBarGraph.js

```diff
@@ -0,0 +1,321 @@
+          return acc
+        },
+        {},
+        filledMachines
+      )
+    )
+  )
+
+  const topMachines = R.take(AMOUNT_OF_MACHINES, devicesByVolume)
+
+  const txClassVolumeByDevice = R.reduce(
+    (acc, value) => {
+      acc[value[0]] = getDeviceVolumeByTxClass(value[0])
+      return acc
+    },
+    {},
+    topMachines
+  )
+
+  const x = d3
+    .scaleBand()
+    .domain(topMachines)
+    .range([GRAPH_MARGIN.left, GRAPH_WIDTH - GRAPH_MARGIN.right])
+    .paddingInner(BAR_PADDING)
+
+  const y = d3
+    .scaleLinear()
+    .domain([
+      0,
+      d3.max(topMachines, d => d[1]) !== 0 ? d3.max(topMachines, d => d[1]) : 50
+    ])
+    .range([GRAPH_HEIGHT - GRAPH_MARGIN.bottom, GRAPH_MARGIN.top])
+
+  const buildXAxis = useCallback(
+    g =>
+      g
+        .attr('class', 'x-axis-1')
+        .attr(
+          'transform',
+          `translate(0, ${GRAPH_HEIGHT - GRAPH_MARGIN.bottom})`
+        )
+        .call(
+          d3
+            .axisBottom(x)
+            .tickFormat(
+              d =>
+                `${R.find(it => it.code === d[0], filledMachines).display ??
+                  ''}`
+            )
+            .tickSize(0)
+            .tickPadding(10)
+        ),
+    [GRAPH_MARGIN, x, filledMachines]
+  )
+
+  const buildXAxis2 = useCallback(
+    g => {
+      g.attr('class', 'x-axis-2')
+        .attr(
+          'transform',
+          `translate(0, ${GRAPH_HEIGHT - GRAPH_MARGIN.bottom})`
+        )
+        .call(
+          d3
+            .axisBottom(x)
+            .tickFormat(d =>
+              R.includes(`ghostMachine`, d[0]) ? `` : `${d[1]} ${currency}`
+            )
+            .tickSize(0)
+            .tickPadding(10)
+        )
+    },
+    [GRAPH_MARGIN, x, currency]
+  )
+
+  const positionXAxisLabels = useCallback(() => {
+    d3.selectAll('.x-axis-1 .tick text').attr('transform', function(d) {
+      const widthPerEntry = (x.range()[1] - x.range()[0]) / AMOUNT_OF_MACHINES
+      return `translate(${-widthPerEntry / 2.25 + this.getBBox().width / 2}, 0)`
+    })
+  }, [x])
+
+  const positionXAxis2Labels = useCallback(() => {
+    d3.selectAll('.x-axis-2 .tick text').attr('transform', function(d) {
+      const widthPerEntry = (x.range()[1] - x.range()[0]) / AMOUNT_OF_MACHINES
+      return `translate(${widthPerEntry / 2.25 - this.getBBox().width / 2}, 0)`
+    })
+  }, [x])
+
+  const buildYAxis = useCallback(
+    g =>
+      g
+        .attr('transform', `translate(${GRAPH_MARGIN.left}, 0)`)
+        .call(
+          d3
+            .axisLeft(y)
+            .ticks(GRAPH_HEIGHT / 100)
+            .tickSize(0)
+            .tickFormat(``)
+        )
+        .call(g => g.select('.domain').remove()),
+    [GRAPH_MARGIN, y]
+  )
+
+  const formatTicksText = useCallback(
+    () =>
+      d3
+        .selectAll('.tick text')
+        .style('stroke', fontColor)
+        .style('fill', fontColor)
+        .style('stroke-width', 0.5)
+        .style('font-family', fontSecondary),
```

Comment:
I would use `font-weight` property here
```suggestion
        .selectAll('.tick text')
        .style('fill', fontColor)
        .style('font-family', fontSecondary)
        .style('font-weight', 'bold'),
```

Reply:
Hmm, it doesn't seem to have the same effect as the spec wants :(